src/extension.ts

```diff
@@ -248,4 +291,12 @@ export async function activate(ctx: vscode.ExtensionContext) {
       runTask(taskItem.task);
     }
   }
+
+  ctx.subscriptions.push(client);
+
+  ctx.subscriptions.push(
+    vscode.commands.registerCommand("mesonbuild.restart", async () => {
+      client.restart();
+    })
+  );
```

Comment:
What is `client` when a binary isn't found?

Reply:
If no binary is found, it basically does nothing.